packages/riipen-ui/src/components/Editor.jsx

```diff
@@ -141,15 +141,20 @@ class Editor extends React.Component {
     mobileControlRow: PropTypes.bool,
 
     /**
-     * Initial content to set in the editor
+     * Initial content to set in the editor.
      */
     initialValue: PropTypes.any,
 
     /**
-     * Function to execute when editor content changes, gets html value of editor
+     * Function to execute when editor content changes, gets html value of editor.
      */
     onChange: PropTypes.func,
 
+    /**
+     * The max height of the Editor text area.
+     */
+    maxHeight: PropTypes.number,
```

Comment:
I don't think `number` is right here. In the example above you used `"200px"` as a value, which is a string. I think we want string here instead. This will cover all cases like "200px", "75%", "5em", "auto", etc.